homework/Selenium_Basics/EpamSiteNavigateTests.cs

```diff
@@ -0,0 +1,49 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using NUnit.Framework;
+
+namespace Selenium_Basics
+{
+    [TestFixture]
+    public class EpamSiteNavigateTests
+    {
+        private IWebDriver _driver;
+
+        [SetUp]
+        public void InicializeBrowser()
+        {
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+        }
+
+        [Test]
+        public void CheckBrowserPageTest()
+        {
+            var link = "https://www.epam.com/";
+
+            _driver.Navigate().GoToUrl(link);
+
+            Assert.That(_driver.Url, Is.EqualTo(link), "Incorrect url is present!");           
+        }
+
+        [Test]
+        public void CheckPageAfterReloadTest()
```

Comment:
same :) name can be more meaningful like CheckWeDoItPageIsOpenedAfterMoveAndReload